### PR TITLE
Remove the custom variant split hack for brexit discount

### PR DESCRIPTION
**Pull request by tavvy**
Also standardise and abstract out approach for use in the future.

### CODE REVIEW

File: server/lib/ab-cohort-discount.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+// Update flags and n-header-footer accordingly
+
+// generic discount mapped from barrierDiscountByCohort MVT
+const discount = {
+	L: .50,
+	S: .25
+}
+
+// brexit offer ids
+const brexitOfferId = {
```
Comment (lc512k) [inline server/lib/ab-cohort-discount.js]: brexit

File: server/lib/ab-cohort-discount.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+	L: 'bb776c53-abdd-280d-4279-cd9aeb0257ff',
+	S: '1dbc248e-b98d-b703-bc25-a05cc5670804'
+}
+
+module.exports = {
+	applyDiscountToBarrierModel: function (barrier, flags) {
+		if (barrier.packages.standard) {
+
+			const percent = 1 - discount[flags.barrierDiscountByCohort];
+			const standardOfferId = brexitOfferId[flags.barrierDiscountByCohort];
```
Comment (lc512k) [inline server/lib/ab-cohort-discount.js]: ^^

File: server/controllers/barriers.js [hunk 3]
```diff
@@ -35,10 +35,7 @@ let barrier = {};
 
 module.exports = function (req, res, next) {
 	let isFragment;
-
-	let mustShowBrexitDiscount;
```
Comment (lc512k) [inline server/controllers/barriers.js]: Great to be getting rid of this with 100% range on`barrierDiscountByCohort`! :+1:

**Comment by tavvy** [comment]
I have left in the `brexit` nomenclature to help keep note of the fact that those offer ids are strictly tied to the brexit offer. Assuming that:
a) that is indeed the case
b) when we need to change to a new offer array we would have `usElectionOfferIds` replace that one, or something like that

**Comment by lc512k** [comment]
Yeah, each offer id is tied to the discount itself, 50%, 25%... LGTM

**Pull request merged**